components/user_settings/security/user_access_token_section/user_access_token_section.jsx

```diff
@@ -4,12 +4,12 @@
 
 import PropTypes from 'prop-types';
 import React from 'react';
-import {FormattedMessage} from 'react-intl';
+import { FormattedMessage } from 'react-intl';
```

Comment:
Nit: would it be ok to set your formatter to match the Mattermost style? A lot of these changes are unrelated to the PR.

Reply:
@nevyangelova thanks. I didn't realize my editor messed up. It's fixed now